### DISCUSSION TITLE
Support all the rook secrets to be stored in Vault

### DISCUSSION THREAD

**Discussion by sunilkpmbl** [rook/rook#7018] [Q&A] [answered]
Hi team,
There was a ticket for moving all the secrets to Vault from Kubernetes secrets. 
[Support all the rook secrets to be stored in Vault #6374](https://github.com/rook/rook/issues/6374)
Is this currently possible? 
If not, can we expect this in an upcoming release? Could you give an expected version and timeline?
Thanks in advance.

**Comment by leseb** [comment, marked as answer]
Hey @sunilkpmbl I believe this requires more investigation and nobody is assigned to that yet. Currently, it is not part of any release plan so we don't have any ETA for this feature.

**Reply by sunilkpmbl** [reply]
Oaky. Thanks @leseb for the quick reply. Since you have added support for Vault as KMS, it will help if you consider this as well. Looking forward for a plan soon. Thanks!
